packages/ui/README.md

```diff
@@ -0,0 +1,8 @@
+# React + Vite
```

Comment:
Can we add some other notes in this README? Some basics for package, a version of what you wrote in the PR works for now 🤷  until we figure out how we want to document it